meson.build

```diff
@@ -131,7 +131,13 @@ else
   dxvk_wsi = get_option('dxvk_native_wsi')
   
   if dxvk_wsi == 'sdl2'
-    lib_sdl2 = cpp.find_library('SDL2')
+    if platform == 'freebsd'
+      # kinda hacky
+      dxvk_include_dirs += ['/usr/local/include']
```

Comment:
You should remove your local hack for this

Reply:
FreeBSD installs all packages to /usr/local. DXVK includes `<SDL2/SDL.h>` which is located in /usr/local/include. Also as I know, SDL2 recommends to use include `<SDL.h>` instead, luckily it will be changed to `<SDL3/SDL.h>` with SDL3 release.

Reply:
How do other meson projects targeting freebsd handle this? We don't have anything like this in gamescope and we support FreeBSD there... at least it builds anyway?

cc: @jbeich

Reply:
See https://github.com/mesonbuild/meson/issues/4468. Besides hardcoding `/usr/local/include` in the source or CI config the value can be derived via  ` get_option('prefix') / 'include'`  or  `dep_displayinfo.get_variable(pkgconfig: 'includedir'))` a la https://github.com/st3r4g/wlanthy/commit/89f2a6a4d0ca.

Note, gamescope uses `dependency('SDL2')` instead of `compiler.find_library('SDL2')` thus ex plicitly passing `/usr/local/include` isn't necessary.

Reply:
Meson have weird dependency handling, if provide `dependency('sdl2')`, it will compile wsi normally but not other libs that depends on it because include path isn't added.